### PR TITLE
disttask: add benchmark to test scheduler overhead

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: ref #49008 

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Review by lance6716 — APPROVED** [review]

File: pkg/disttask/framework/integrationtests/bench_test.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+)
+
+// test overhead when starting multiple schedulers
+//
+// make failpoint-enable
+// GOOS=linux GOARCH=amd64 go test -tags intest -c -o bench.test ./pkg/disttask/framework/integrationtests
+// make failpoint-disable
+//
+// bench.test -test.v -run ^$ -test.bench=BenchmarkSchedulerOverhead --with-tikv "upstream-pd:2379?disableGC=true"
+func BenchmarkSchedulerOverhead(b *testing.B) {
```
Comment (hawkingrei) [inline pkg/disttask/framework/integrationtests/bench_test.go:56]: You should add this bench at the dailybench.

Comment (D3Hunter) [reply pkg/disttask/framework/integrationtests/bench_test.go:56]: it's for local performance test

**Pull request merged**